bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/dto/onecall/Daily.java

```diff
@@ -70,6 +73,30 @@ public void setSunrise(int sunrise) {
         this.sunrise = sunrise;
     }
 
+    public int getMoonrise() {
+        return moonrise;
+    }
+
+    public void setMoonrise(int moonrise) {
+        this.moonrise = moonrise;
+    }
+
+    public int getMoonset() {
+        return moonset;
+    }
+
+    public void setMoonset(int moonset) {
+        this.moonset = moonset;
+    }
+
+    public double getMoon_phase() {
```

Comment:
Please follow the Java naming conventions (https://java.about.com/od/javasyntax/a/nameconventions.htm):

```suggestion
    public double getMoonPhase() {
```